src/common/signalk/SKJSONVisitor.cpp

```diff
@@ -30,16 +30,47 @@
 
 #include "SKJSONVisitor.h"
 
+void SKJSONVisitor::processSource(const SKSource &source, JsonObject &sourceObject) {
+  sourceObject["label"] = source.getLabel().c_str();
+
+  switch (source.getInput()) {
```

Comment:
Any chance the input pin ID could be included in the signal K output JSON (maybe in the metadata)?